lib/pi_piper/driver.rb

```diff
@@ -0,0 +1,114 @@
+module PiPiper
+  class Driver
+    
+    VERSION = '0.1.0'
+    ALLOW = []
+    API_VERSION = '0.1.0'
```

Comment:
I get your point. But do we want to lock the version number between gems ?
On Mar 9, 2016 4:57 AM, "Zshawn Syed" <notifications@github.com> wrote:

> In lib/pi_piper/driver.rb
> <https://github.com/jwhitehorn/pi_piper/pull/66#discussion_r55469540>:
>
> > @@ -0,0 +1,114 @@
> > +module PiPiper
> > +  class Driver
> > +
> > +    VERSION = '0.1.0'
> > +    ALLOW = []
> > +    API_VERSION = '0.1.0'
>
> What is this used for? The PiPiper version should be sufficient since
> this driver will be in lockstep with those releases. If someone needs a
> specific version, they can manage which version of PiPiper they want.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jwhitehorn/pi_piper/pull/66/files#r55469540>.
>